cms/jinja2/templates/pages/analysis_page.html

```diff
@@ -4,19 +4,20 @@
 {% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
 {% from "components/text-indent/_macro.njk" import onsTextIndent %}
 
+
 {% block header_area %}
-    <div class="analysis-header">
+    <div class="analysis-header {% if page.updates %}analysis-header--flush{% endif %}">
         <div class="ons-container">
             {% include "templates/components/navigation/breadcrumbs.html" %}
-            <p class="analysis-header__doc-type">{{ _("Analysis") }}</p>
+            <p class="analysis-header__doc-type ons-u-fs-l">{{ _("Analysis") }}</p>
```

Comment:
The designs (and prototype) now have 

```suggestion
            <p class="analysis-header__doc-type ons-u-fs-l">{{ _("Statistical article") }}</p>
```

should we stick to it?

Reply:
I was going by the figma file - and I think I recall a November prototype conversation where it was agreed it should be 'Statistical article' - but I will check.

Reply:
Sorry - I misunderstood the diff there, and thought I already had 'Statistical article' as the text - however @MebinAbraham has suggested on slack that we leave it as 'Analysis' and have a separate card for updating to 'Statistical article'.

Reply:
Works for me